src/server/pfs/server/master.go

```diff
@@ -38,16 +38,33 @@ const (
 	masterLockPath = "pfs-master-lock"
 )
 
-func (d *driver) master(ctx context.Context) {
+type Master struct {
+	env Env
+
+	d *driver
```

Comment:
[nit] I prefer using entire struct names as fields because it helps to enhance searchability. I'm thinking of grepping for 'driver' for instance to try to find all places where we instantiate the driver or access/use the master's driver.